website/client/js/controllers/memberModalCtrl.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
 habitrpg
-  .controller("MemberModalCtrl", ['$scope', '$rootScope', 'Members', 'Shared', '$http', 'Notification', 'Groups', 'Chat', '$controller', 'Stats',
-    function($scope, $rootScope, Members, Shared, $http, Notification, Groups, Chat, $controller, Stats) {
+  .controller("MemberModalCtrl", ['$scope', '$rootScope', 'Members', 'Shared', '$http', 'Notification', 'Groups', 'Chat', '$controller', 'Stats', 'Content', 'Costume',
+    function($scope, $rootScope, Members, Shared, $http, Notification, Groups, Chat, $controller, Stats, Content, Costume) {
```

Comment:
Oh! Probably.
On Wed, Aug 24, 2016 at 6:53 PM Kaitlin Hipkin <notifications@github.com>
wrote:

> In website/client/js/controllers/memberModalCtrl.js
> <https://github.com/HabitRPG/habitrpg/pull/7768#discussion_r76160549>:
>
> > @@ -1,8 +1,8 @@
> >  "use strict";
> >
> >  habitrpg
> > -  .controller("MemberModalCtrl", ['$scope', '$rootScope', 'Members', 'Shared', '$http', 'Notification', 'Groups', 'Chat', '$controller', 'Stats',
> > -    function($scope, $rootScope, Members, Shared, $http, Notification, Groups, Chat, $controller, Stats) {
> > +  .controller("MemberModalCtrl", ['$scope', '$rootScope', 'Members', 'Shared', '$http', 'Notification', 'Groups', 'Chat', '$controller', 'Stats', 'Content', 'Costume',
> > +    function($scope, $rootScope, Members, Shared, $http, Notification, Groups, Chat, $controller, Stats, Content, Costume) {
>
> Well, you added it, but it doesn't look like you're using it. I think it
> can be removed. Perhaps it got overlooked in one of your refactors?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7768/files/291cd5d35d55e3de3a33586a2afedcb832c77fd0#r76160549>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_84nI3YRTIFiw_2KkJsLn2OiJ_jF7ks5qjNlmgaJpZM4JIjd_>
> .
>